### PR TITLE
Fix Circular dependencies in device registry.

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**
- Removes circular dependency in Device registry.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-115](https://airqoteam.atlassian.net/browse/AN-115)
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**
- Checkout this branch.
- Run `npm install` and `npm run dev-mac`.
- There shouldn't be any circular dependency warnings in the console log.

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/device-registry/utils/create-monitor.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ const logger = log4js.getLogger(
 
 const qs = require("qs");
 const QRCode = require("qrcode");
-const { kafkaProducer } = require("../config/kafkajs");
```
Comment (Baalmart) [inline src/device-registry/utils/create-monitor.js:21]: Thanks @BenjaminSsempala , could you add some guiding PR review comments about this deletion?

File: src/device-registry/utils/create-event.js [hunk 2]
```diff
@@ -18,7 +18,6 @@ const { getDevicesCount, list, decryptKey } = require("./create-monitor");
 const HTTPStatus = require("http-status");
 const redis = require("../config/redis");
 const axios = require("axios");
-const { kafkaConsumer } = require("../config/kafkajs");
```
Comment (Baalmart) [inline src/device-registry/utils/create-event.js:21]: Thanks @BenjaminSsempala , could you add some guiding PR review comments about this deletion?

File: src/device-registry/config/constants.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const mongoose = require("mongoose");
 const ObjectId = mongoose.Types.ObjectId;
-const { generateDateFormatWithoutHrs } = require("../utils/date");
+const log4js = require("log4js");
+const logger = log4js.getLogger(`${this.ENVIRONMENT} -- constants-config`);
```
Comment (Baalmart) [inline src/device-registry/config/constants.js:4]: Thanks @BenjaminSsempala 

Does `this.ENVIRONMENT` work? Have you tested it out?

Thanks!

**Review by Baalmart — COMMENTED** [review]
Thanks @BenjaminSsempala ....this is much appreciated.

I have just left a few comments for your review.

File: src/device-registry/utils/create-event.js (hunk 2 above)
Comment (BenjaminSsempala) [inline src/device-registry/utils/create-event.js:21]: This removes an unused import.

File: src/device-registry/utils/create-monitor.js (hunk 1 above)
Comment (BenjaminSsempala) [inline src/device-registry/utils/create-monitor.js:21]: This removes an unused import.

File: src/device-registry/config/constants.js (hunk 3 above)
Comment (BenjaminSsempala) [reply src/device-registry/config/constants.js:4]: Yes @Baalmart , It seemed to work fine. 
Although when it seems some new circular dependencies have emerged on staging. Let me try to address those too.

Comment (Baalmart) [reply src/device-registry/config/constants.js:4]: Alright, thanks! It is becoming a real nightmare indeed!

**Pull request merged**